Description: Default to PIC on arm
 Otherwise the armhf build fails now that PIE is enabled by default.
 TODO: there might be a better way to do this.
Author: Ximin Luo <infinity0@debian.org>
Forwarded: TODO
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/utils/clflags.ml
+++ b/utils/clflags.ml
@@ -147,6 +147,7 @@
 
 let pic_code = ref (match Config.architecture with (* -fPIC *)
                      | "amd64" -> true
+                     | "arm" -> true
                      | _       -> false)
 
 let runtime_variant = ref "";;      (* -runtime-variant *)
--- a/man/ocamlopt.m
+++ b/man/ocamlopt.m
@@ -702,10 +702,10 @@
 Select the floating-point hardware
 .TP
 .B \-fPIC
-Generate position-independent machine code.
+Generate position-independent machine code.  This is the default.
 .TP
 .B \-fno\-PIC
-Generate position-dependent machine code.  This is the default.
+Generate position-dependent machine code.
 .TP
 .B \-fthumb
 Enable Thumb/Thumb-2 code generation
